sos/plugins/gluster_block.py

```diff
@@ -43,3 +40,4 @@ def setup(self):
                 self.add_copy_spec(f, limit)
         else:
             self.add_copy_spec("/var/log/gluster-block")
+
```

Comment:
Other than the remaining `targetcli` line these changes look fine, once they are folded into the original commit and the branch updated.